advanced/external_references/README.md

```diff
@@ -259,6 +259,17 @@ end
 Add a new input parameter and a new output parameter *color* as shown above. In the *run()*
 function, just assign the input color to the output color.
 
+In order for the LUA script and LUA interface to work together, you need to add a line for the
+parameter *color* in the interface, as well.
+
+```
+function interface(INOUT)
```

Comment:
```suggestion
function interface(IN, OUT)
```

Reply:
please ignore, your version is the correct one